pom.xml

```diff
@@ -82,22 +83,14 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
       <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>javax.servlet</groupId>
-          <artifactId>servlet-api</artifactId>
```

Comment:
`junit` dropped the dependency to `servlet-api` a long, long time ago. No need to exclude it.